### PR TITLE
[IoTHub] Command "az iot hub create" improvement

### PR DESCRIPTION
**Pull request by anusapan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by anusapan** [comment]
@qwordy Could you review this PR?

**Comment by yonzhan** [comment]
@qwordy please help with this PR.

File: src/azure-cli/azure/cli/command_modules/iot/_validators.py [hunk 1]
```diff
@@ -15,3 +15,63 @@ def validate_policy_permissions(ns):
...
+def validate_fileupload_notification_max_delivery_count(ns):
+    if (ns.fileupload_notification_max_delivery_count and
+            ns.fileupload_notification_max_delivery_count not in range(1, 101, 1)):
+        raise ArgumentError(None, 'Please specify the number of retry from 1 to 100 only to deliver a message')
+
+
+def validate_fileupload_notification_ttl(ns):
+    if (ns.fileupload_notification_ttl and
+            ns.fileupload_notification_ttl not in range(1, 49, 1)):
+        raise ArgumentError(None, 'Please specify the period of time from 1 to 48 for which a file upload'
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/iot/_validators.py]: Add unit, which is hour.

**Comment by qwordy** [comment]
ERROR: ************* Module azure.cli.command_modules.iot.custom
src/azure-cli/azure/cli/command_modules/iot/custom.py:1:0: C0302: Too many lines in module (1034/1000) (too-many-lines)
src/azure-cli/azure/cli/command_modules/iot/custom.py:368:0: R0914: Too many local variables (29/25) (too-many-locals)
src/azure-cli/azure/cli/command_modules/iot/custom.py:1:0: C0302: Too many lines in module (1034/1000) (too-many-lines)
src/azure-cli/azure/cli/command_modules/iot/custom.py:368:0: R0914: Too many local variables (29/25) (too-many-locals)

File: src/azure-cli/azure/cli/command_modules/iot/_validators.py (hunk 1 above)
Comment (anusapan) [reply src/azure-cli/azure/cli/command_modules/iot/_validators.py]: fixed

**Pull request merged**